packages/openapi-generator/src/generator.ts

```diff
@@ -101,6 +102,26 @@ async function generateSDKSources(
   }
 }
 
+async function createApis(
+  serviceDir: string,
+  openApiDocument: OpenApiDocument
+) {
+  return openApiDocument.tags.map(tag => {
+    const matchedOperations = openApiDocument.operations.filter(o =>
+      o.tags?.includes(tag)
+    );
```

Comment:
What happens if the operation has no tags? Please see my comment in document-converter regarding this.

Reply:
Consider adjusting the operation type to `tags: string[]` instead of `tags?: string[]` based on the comment above.

Reply:
@marikaner , the default tag is added when tags are missing.
However, I still use the original openapi type, which contains `tags?: string[]`, but I alway use `!` to tell "I'm sure now `tags` is there."
I know it's confusing since `tags` is not optional anymore, but if you change the type, you have to differentiate between the old one (before convert) and new one (after convert) and really create a new type.
Since this is the first time we see the use case of the new type, if this can be avoided, I would still keep using the original type.
What's your feeling?

Reply:
discussed